src/mcts/node.cc

```diff
@@ -230,6 +230,17 @@ void Node::MakeTerminal(GameResult result) {
   }
 }
 
+void Node::AllowBranching() {
```

Comment:
`MakeNotTerminal` ? `UnmakeTerminal` ? `UndoMakeTerminal`?

Reply:
Renamed to `MakeNotTerminal`